src/agent/rustjail/src/lib.rs

```diff
@@ -215,8 +215,8 @@ fn idmaps_grpc_to_oci(ims: &[grpcLinuxIDMapping]) -> Vec<ociLinuxIDMapping> {
 }
 
 use oci::{
-    LinuxBlockIO as ociLinuxBlockIO, LinuxBlockIODevice as ociLinuxBlockIODevice,
-    LinuxCPU as ociLinuxCPU, LinuxDeviceCgroup as ociLinuxDeviceCgroup,
+    LinuxBlockIo as ociLinuxBlockIo, LinuxBlockIoDevice as ociLinuxBlockIoDevice,
```

Comment:
I think the OciLinuxBlockIo is better than ociLinuxBlockIo and the same for others.

Reply:
They are still types, so yes, Rust conventions would say they should be LikeThis not likeThis.  However, that problem exists with a bunch of the imports here that aren't touched by this change, so we really want to change them all together for consistency.

I don't think we want to delay this fix for that additional cleanup.

Reply:
Actually, I take that back, it bugged me so I fixed it up and added it to this series.

Reply:
cool